homeworks/05-redux-basic/src/combine-reducers.ts

```diff
@@ -0,0 +1,17 @@
+type Reducer<S, A> = (state: S | undefined, action: A) => S
+type ReducerMap<S, A> = {
+    [K in keyof S]: Reducer<S[K], A>
+}
+
+export const combineReducers = <S, A>(
```

Comment:
Not quite sure why this is needed - Redux library already has this function `combineReducers`.

Reply:
This was the other homework. I just did it together to make sure my `combineReducers` worked as intended.